src/quick-start.md

```diff
@@ -166,27 +163,41 @@ EOF
 ```
 
 This will produce the following output:
-`clusteradmissionpolicy.policies.kubewarden.io/privileged-pods created`
+```shell
+clusteradmissionpolicy.policies.kubewarden.io/privileged-pods created
+```
+
+When a  `ClusterAdmissionPolicy` is defined, the status is set to `pending`, and it will force a rollout of the targeted `PolicyServer`. In our example, it's the `PolicyServer` named `default`. You can monitor the rollout by running the following command:
+
+```shell
+kubectl get clusteradmissionpolicy.policies.kubewarden.io/privileged-pods
+```
 
-Defining the `ClusterAdmissionPolicy` will set its status to `pending`, and it will lead to a rollout of the `PolicyServer` named `default`. Once the new policy is ready to be served, the `kubewarden-controller`
-will register a [ValidatingWebhookConfiguration](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#validatingwebhookconfiguration-v1-admissionregistration-k8s-io)
-object.
+You should see the following output:
 
-`ClusterAdmissionPolicy` status will be set to `active` once all the instances of Deployment rollout of the `policy-server` is done. The
-`ValidatingWebhookConfiguration` can be shown with:
+```shell
+NAME              POLICY SERVER   MUTATING   STATUS
+privileged-pods   default         false      pending
+```
+
+Once the new policy is ready to be served, the `kubewarden-controller` will register a [ValidatingWebhookConfiguration](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#validatingwebhookconfiguration-v1-admissionregistration-k8s-io) object.
+
+The `ClusterAdmissionPolicy` status will be set to `active` once the Deployment is done for every `policy-server` instances. The `ValidatingWebhookConfiguration` can be shown with the following command:
 
 ```shell
 kubectl get validatingwebhookconfigurations.admissionregistration.k8s.io -l kubewarden
 ```
 
-Which will output something like
+You should see the following output:
 
 ```
-NAME              WEBHOOKS   AGE     STATUS
-privileged-pods   1          9s      active
+NAME              WEBHOOKS   AGE
+privileged-pods   1          9s
 ```
 
-Let's try to create a Pod with no privileged containers:
+Once the `ClusterAdmissionPolicy` is active and the `ValidatingWebhookConfiguration` is registered, you can test the policy.
+
+First, let's create a Pod without any Container `privileged` flag:
```

Comment:
Nit:

```suggestion
First, let's create a Pod with a Container *not* in `privileged` mode:
```

Flag would be `--privileged`, as when one uses a cli such as docker. Here we are telling Kubernetes to start a container without privileged mode (see wording for example in https://kubernetes.io/docs/concepts/policy/pod-security-policy/#privileged)